pkg/ingester/ingester.go

```diff
@@ -719,6 +719,36 @@ type pushStats struct {
 	perMetricSeriesLimitCount int
 }
 
+// AcceptPushRequest returns error if push request cannot be accepted at the moment.
+// When this method returns error, it increases relevant metrics.
+func (i *Ingester) AcceptPushRequest() error {
+	if err := i.checkRunning(); err != nil {
+		return util_log.DoNotLogError{Err: err}
```

Comment:
wrapping this in `DoNotLogError` probably won't do much because the code which checks for this interface is an interceptor. This error shouldn't be logged anyways I think (?)

Reply:
That's a good point, you're right on both counts. I'll add a comment explaining why we skip wrapper here.